independent-publisher-connectors/HTTP Garden/apiDefinition.swagger.json

```diff
@@ -0,0 +1,97 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "http garden",
+    "description": "Gardening for every Hyper Text Transfer Protocol response status code. Provides a image for each status code.",
+    "version": "1.0",
+    "contact": {
+      "name": "Troy Taylor",
+      "url": "https://www.hitachisolutions.com",
+      "email": "ttaylor@hitachisolutions.com"
+    }
+  },
+  "host": "http.garden",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/{code}.{file}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "$content-type": {
+                  "type": "string",
+                  "description": "The content file type.",
+                  "title": "Content Type"
+                },
+                "$content": {
+                  "type": "string",
+                  "description": "The file content.",
+                  "title": "Content"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get status code",
+        "description": "Retrieve a image based on the requested status code.",

```

Comment:
@troystaylor please make sure that everything is grammatically correct and is free of typographical errors.